tests/compile-fail/bool_comparison.rs

```diff
@@ -0,0 +1,12 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#[allow(needless_bool)]
+#[deny(bool_comparison)]
+fn main() {
+    let x = true;
+    if x == true { true } else { false }; //~ERROR you can simplify this boolean comparison to `x`
```

Comment:
Change all of these test cases to `{ "yes" } else { "no" }` to get rid of the needless_bool error.